null_safety_examples/cookbook/persistence/reading_writing_files/test/widget_test.dart

```diff
@@ -24,3 +32,19 @@ void main() {
     // #enddocregion setUpAll
   });
 }
```

Comment:
`flutter test` for just this existing test was resulting in - `No tests ran.`, which was failing the `tool/check-code.sh --null-safety`